tools/v8_gypfiles/toolchain.gypi

```diff
@@ -273,6 +273,11 @@
           }],
         ],
       }],
+      ['v8_target_arch=="riscv64"', {
+        'defines': [
+          'V8_TARGET_ARCH_RISCV64',
```

Comment:
V8 defines more macros: https://github.com/nodejs/node/blob/d7b86385821f117eecc11a8bd55f9a4d615a5eb0/deps/v8/BUILD.gn#L934-L940

Aren't they needed for Node.js too?

Reply:
If delte it, riscv64 build will failed.

Reply:
I didn't suggest to remove it. I'm asking whether we should add the other ones that are in V8

Reply:
Oh sorry, i see you. Do you mean the gypfiles should been generated by v8 ?

Reply:
I can't find gypfile in v8/master. Can you tell me how dou you update v8_gypfile?

Reply:
No, gypfiles are maintained by us, and we need to change them manually.
Let me rephrase my question:

This pull request only adds `"V8_TARGET_ARCH_RISCV64"`, but in V8's `BUILD.gn`, they also define `"__riscv_xlen=64"` and `"CAN_USE_FPU_INSTRUCTIONS"`. Why are these not ported to the gypfile?

Reply:
Thanks your remind , i had fixed it.